templates/case-study/unibap-space-computing.html

```diff
@@ -0,0 +1,255 @@
+      Unibap selected Ubuntu as the pre-installed OS that would accompany their hardware, providing a robust and accessible foundation for time-critical space computing.
+    {%- endif -%}
+    {%- if slot == 'highlights_item_content_2' -%}
+      With 10 years of support through Ubuntu Pro, the OS enables Unibap to meet the needs of long lifespan satellite-based projects.
+    {%- endif -%}
+    {%- if slot == 'highlights_item_content_3' -%}
+      Most recently, a single Unibap computer powered by Ubuntu was used to run 40 different experiments from a satellite, across a range of use cases.
+    {%- endif -%}
+  {%- endcall %}
+
+  {%- call(slot) image_wrapper() -%}
+    {%- if slot == 'image_content' %}
+      {{ image(url="https://assets.ubuntu.com/v1/b68b2931-moon.png",
+            alt="",
+            width="1848",
+            height="771",
+            hi_def=True,
+            loading="lazy",
+            attrs={"class": "p-image-container__image"}) | safe
+      }}
+    {%- endif -%}
+  {%- endcall -%}
+
+  {%- call(slot) text(title="A space platform that creates space for innovation") -%}
+    {%- if slot == 'content' -%}
+      <p>
+        Satellites are the driving force behind much of modern life. Navigation, communications, and disaster monitoring are just some of the use cases that have become so ubiquitous that we often forget they occur “off-planet”. However, these technological advancements are often supported by satellite hardware that predates them by 20-30 years, which presents a challenge for organizations looking to pioneer intensive, novel use cases like AI and edge computing in space.
+      </p>
+      <p>
+        Unibap's range of hardware solutions are designed to prioritize compatibility by default, providing a pre-assembled platform that is engineered for the idiosyncrasies of both the host satellites, and the physical challenges of space. Unibap selected Ubuntu as the operating system that could best match their hardware in terms of robustness and provide an accessible platform for developers to pioneer novel use cases in space.
+      </p>
+    {%- endif -%}
+  {%- endcall -%}
+
+  {%- call(slot) text(title="Challenge") -%}
+    {%- if slot == 'content' -%}
+      <p>
+        Satellite technology is somewhat paradoxical. It represents technological progress and serves as the foundation for many of the innovations we continue to see. At the same time, satellite hardware is ageing, with many satellites exceeding their intended design life. This means that creativity is required to pioneer new use cases and sustainably introduce new innovations, like edge computing and AI, to the satellite context.
+      </p>
+      <p>
+        Unibap is tackling this challenge head on. Their mission is to deliver trustworthy innovation in space through their suite of hardware and software solutions which are designed to bridge the gap between cutting-edge advancements and aging satellite hardware.
```

Comment:
```suggestion
        Unibap is tackling this challenge head on. Their mission is to deliver trustworthy innovation in space through their suite of hardware and software solutions which are designed to bridge the gap between cutting-edge advancements and aging satellite hardware.
```
We should choose one spelling 'aging' or 'ageing'. I left a note in the copydoc